dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -93,6 +93,17 @@ in `src/airflow_breeze/utils/packages.py` and run the `prepare-provider-document
 command with the `--only-min-version-update` flag. This will only update the min version in
 the `__init__.py` files and package documentation without bumping the provider versions.
 
+
+Note: Sometimes we are releasing a subset of providers and would not want to add the
+list of these providers to every breeze command we run, specifically:
```

Comment:
We have several occurrences in the guide were we have:
`If we want to just release some providers you can release them using package names:`
or similar we should modify these to use the new var

Reply:
I think we should be supporting both the old and new technique. So that the person running the script has 2 options in case one doesn't work out. WDYT @eladkal ?

Reply:
yes. I think the old way is nice because of autocomplete support and it works nicer for someone who just want to build one package for example. I often tell people if they want to test main version of provider, that they use `breeze release-management prepare-provider-packages <PACKAGE> --skip-tag-check` to do so.